packages/block-library/src/image/view.js

```diff
@@ -466,6 +443,14 @@ const { state, actions, callbacks } = store(
 				}
 			`;
 			},
+			setScreenReaderText() {
+				const { ref } = getElement();
+				ref.textContent = state.overlayEnabled
+					? `Image ${ state.currentImageIndex + 1 } of ${
+							state.images.length
+					  }`
```

Comment:
Is there a way to make front-end text translatable in the current WordPress/Gutenberg?

Reply:
fixed in b9443550f140e44df1dab807c71becc047806bc7